app/controllers/events_controller.rb

```diff
@@ -9,7 +9,7 @@ def show
 
   def random_attendee
     attendes = meetup.attendence(urlname, event_id)
-    person = attendes.shuffle.pop[:member]
+    person = attendes.sample.member
```

Comment:
@Vitormdias I agree, we can just call `randon_attendee` from the controller. But I think it's better to keep the `attendance` method on the service, as we can call it from within the `randon_attendee` to get the participants.